### PR TITLE
Build the project with Webpack

### PR DESCRIPTION
**Pull request by martinnajjar12**
### User Interface

In this branch, I've created a simple navigation bar which allows you to navigate between the links easily because of the power of Webpack and modules of JavaScript.

The used dependencies in this project are: 
- Webpack
- Babel
- Style loader
- CSS loader
- File loader
- Image webpack loader

### CODE REVIEW

File: src/scripts/contactPage.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+function contactPage() {
```
Comment (whiz25) [inline src/scripts/contactPage.js]: - one of the objectives of this project is on use of ES6 syntax. Convert all your code to ES6
```javascript
const contactPage = () {}
```

File: src/scripts/recipesPage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+function recipesPage() {
```
Comment (whiz25) [inline src/scripts/recipesPage.js]: - convert to ES6 syntax

File: src/scripts/nav.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  container.className = 'container';
+  const navbarBrand = document.createElement('a');
+  navbarBrand.className = 'navbar-brand';
+  navbarBrand.setAttribute('href', '/');
+  navbarBrand.textContent = 'Saj Al-Reef';
+  const navList = document.createElement('ul');
+  navList.className = 'd-flex';
+  navList.appendChild(home);
+  navList.appendChild(recipes);
+  navList.appendChild(contact);
```
Comment (whiz25) [inline src/scripts/nav.js:17]: - use a `tab menu` to render the elements not a normal `navbar`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,77 @@
...
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Node.js
+
+### Setup
```
Comment (whiz25) [inline README.md:37]: - How does one clone the repo? Add detailed instructions on how to.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Well done. The project is moving in the right direction: app is deployed and linter errors don't exist.
We however need to fix some crucial features before merging this PR.
- [x] add an entry point file `index.js` at the root of `src` folder. This file will be used to render the final app logic to the browser. All modules will be imported here.
- [x] read my comments for suggestions

### Martin Afani

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great job implementing the suggested changes.

**You may move on.**

_Good Luck 
Simon Wathigo_

**Pull request merged**